app/src/main/java/com/codesoom/assignment/auth/dto/SessionResponseData.java

```diff
@@ -0,0 +1,18 @@
+package com.codesoom.assignment.auth.dto;
+
+import lombok.Getter;
+
+/**
+ * 사용자인증 응답 객체.
+ */
+@Getter
+public class SessionResponseData {
+    /**
+     * 사용자 인증 토큰.
+     */
+    private String accessToken;
```

Comment:
`final`을 붙여주면 더 좋겠군요.
```suggestion
    private final String accessToken;
```

Reply:
감사합니다.